_episodes/10-collaborating.md

```diff
@@ -87,20 +102,40 @@ Delta compression using up to 4 threads.
 Compressing objects: 100% (2/2), done.
 Writing objects: 100% (3/3), 306 bytes, done.
 Total 3 (delta 0), reused 0 (delta 0)
-To https://github.com/vlad/planets.git
-   9272da5..29aba7c  master -> master
+To https://github.com/ccline/planets.git
+   9272da5..29aba7c  standardize-dates -> standardize-dates
 ~~~
 {: .output}
 
 Note that we didn't have to create a remote called `origin`: Git uses this
 name by default when we clone a repository.  (This is why `origin` was a
 sensible choice earlier when we were setting up remotes by hand.)
 
-Take a look to the Owner's repository on its GitHub website now (maybe you need
-to refresh your browser.) You should be able to see the new commit made by the
-Collaborator.
+Now the Collaborator should open their browser to the Owner's repository.
+
+We're going to create a [Pull Request](https://help.github.com/articles/about-pull-requests/).
+
+You should see a highlighted option noting the new branch and an option to *Compare & pull request*
+
+![Option to create pull request from branch](../fig/github-create-pr.png)
+
+Here you can add additional information to your Pull Request such as reasoning behind your changes. When you are happy
+with the title and description, click on the button to *Create pull request*.
 
-To download the Collaborator's changes from GitHub, the Owner now enters:
+Notice that GitHub will automatically select the branch to merge your Pull Request into. This is usually `master` and
+exactly what you want. However, you can change this to any existing branch in the repository.
+
+The Collaborator's Pull Request will allow the Owner to:
+- review and accept the proposed changes
+- perform a merge, just like we did in the Merging branches lesson
+
+The Owner should now review the Pull Request and [Merge](https://help.github.com/articles/merging-a-pull-request/) it.
```

Comment:
Do all these terms need to be capitalized?

Reply:
I'm keeping with the Owner/Collaborator pattern they have. I do think it helps make it a bit more obvious who is suppose to do what. but no don't think Pull Request and Merge need to be. I'll clean that up. Thanks.